### PR TITLE
New version: Amazon.WorkspacesClient version 5.19.0

### PR DESCRIPTION
**Pull request by PckgrBot**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141903)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @PckgrBot,

This manifest adds Apps and Features entries that aren't present in previous PR versions. These entries should be added to the previous versions, or removed from this version.

(Automated response - build 857.)

**Comment by stephengillie** [comment]
From https://github.com/microsoft/winget-pkgs/pull/141633:
> @ stephengillie Any idea what is going on with this package? It has deleted the latest version before adding an updated version on several occasions...? Right now the latest version on Winget is 3.1.10 but on the Amazon site it is up to 5.19.0
> 
> In the past it has deleted 5.16.0 5 days before adding 5.17.0, same with 5.8.0 and 5.9.0
> 
> It keeps deleting the latest before it adds the new which means the only version available in Winget is a 2.5 year old 3.1.10 for several days

Hi @th3nme,

It appears that there are several things happening: 
- New process is meant to prevent this by holding a removal PR until the complementary addition PR is submitted. 
  - I've been giving this process a timeout of 3 days. So if an addition PR hasn't come up by then, I'll merge anyway to remove from the repo. After this, I'll increase to 7. 
- Very old version should be removed from the repo. 
- Apps And Features error here is incorrect, as all 3 versions have them:  
  - [`3.1.10`](https://github.com/microsoft/winget-pkgs/blob/master/manifests/a/Amazon/WorkspacesClient/3.1.10/Amazon.WorkspacesClient.installer.yaml) very old version has them at the installer level.
  - [`5.18.0`](https://github.com/microsoft/winget-pkgs/pull/141633/files) previous (removed) version had these at the root level, with ProductCode specified twice.
  - [`5.19.0`](https://github.com/microsoft/winget-pkgs/pull/141903/files) current (this PR) version has these at the root level, also with ProductCode specified twice. 
  - Guessing this is due to `3.1.10` having it at the installer level. 

To summarize: This PR should merge, I should wait a week before closing removals (and check for situations like this first) and a PR to remove `3.1.10` should be created.

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
[Policy] Reset Feedback

**Review by stephengillie — APPROVED** [review]
